Makefile

```diff
@@ -114,6 +127,12 @@ move-other:
 	mv ${PACKAGE_PREFIX}/${PACKAGE_MAJOR}/api .
 	mv ${PACKAGE_PREFIX}/${PACKAGE_MAJOR}/git_push.sh .
 
+test:
+	go test -v ./...
+
+stage:
+	test -d .git && git add --intent-to-add README.md docs ${PACKAGE_PREFIX} go.mod go.sum
+
 # https://github.com/OpenAPITools/openapi-generator/issues/741#issuecomment-569791780
 remove-dupe-requests: ## Removes duplicate Request structs from the generated code
```

Comment:
This is _kind of_ out of scope here because it's a task that isn't mentioned anywhere currently, but I think this task could be removed.  Based on the comment before it, this task is meant to resolve an issue in `openapi-generator`, which this SDK doesn't use (at least not yet).  Even if you are evaluating `openapi-generator`, I would recommend starting with whatever the generator produces, rather than using this (or any other) workaround tasks.  The generator has gone through many releases since the referenced issue was opened, and may have options/properties that can be set in order to more directly/correctly fix issues with generated code.

Reply:
I'm with you on this one! I'd like to remove as much as possible about what isn't used. Happy to toss this one as well.

Reply:
File this under 'future consideration'..

I've been looking forward to an illustrious future where all of the various codegen SDK repos share a common Makefile and benefit from updates from one repo landing in another. I realize we are a good bit away from that, with the current state being a copy/paste when creating a new repo and then slowly drifting away.

 Some thoughts on how to keep the raft together:
- _equinix-labs/openapi-generated-sdk_
   Create a base repo from which all others are templated.

   **Pro**: This would give the repos a common and visible heritage.
   **Con**: 
     * The ship has sailed for existing repos. We could put the relationship in place on a new repo and force push. Git would be fine, but our issue history would be lost. GitHub support may be able to create the metadata linkage for us without breaking history.
     * There is no code dependency, no git association.
  
- _tasks / common.mk Submodule_:
  Source common Makefile components from an external source.  Example: https://github.com/upbound/build#build and https://github.com/crossplane/crossplane/blob/master/Makefile

   **Pro**: No tight dependency, but a common set of centrally maintained codegen helpers.
   **Con**: we will forever need to clone SDKs with `--submodule` and then maintain them.  
- _tasks / common.mk Fetched_:
  Source common Makefile components from an external source. Similar to the Submodule approach, but using `curl equinix-labs/openapi-generated-sdk/.../latest/common.mk` for updates.

   **Pro**: No tight dependency, but a common set of centrally maintained codegen helpers.
   **Con**: we'll have to update the version in the `curl` and commit the changes, but this could be define as a task in common.mk (`make latest-mk`).
  
I'm sure there are other ways that go beyond accepting our current approach with Makefiles and multiple repositories.  The original hope was to have a single Equinix repo per language, or even a single repo for all generated SDKs.

There is more we can do to improve the current process without resetting. Using a central repo as a holding ground for SDK common tooling including tasks (Makefile includes), Github actions, and OAS tools (patchers / flatteners / splitters) is just one approach.